upup/models/cloudup/_aws/topologies/_topology_private/bastion.yaml

```diff
@@ -0,0 +1,121 @@
+# ---------------------------------------------------------------
+#
+# Private Network Topology in AWS
+#
+# Inspired by https://github.com/kubernetes/kops/issues/428
+#
+# This will handle deploying k8s instance into a private subnet.
+#
+# Where a private subnet means:
+# A subnet doesn't have a route to the Internet gateway.
+#
+# Utility Subnet - A subnet that is used to bridge a private
+# subnet to a public subnet with a NGW
+#
+# ---------------------------------------------------------------
+
+{{ if WithBastion }}
+
+# ---------------------------------------------------------------
+# Security Group - Bastion
+#
+# The security group that the bastion lives in
+# ---------------------------------------------------------------
+securityGroup/bastion.{{ ClusterName }}:
+  vpc: vpc/{{ ClusterName }}
+  description: 'Security group for bastion'
+  removeExtraRules:
+  - port=22
+securityGroupRule/bastion-egress:
+  securityGroup: securityGroup/nodes.{{ ClusterName }}
+  egress: true
+  cidr: 0.0.0.0/0
+securityGroupRule/all-node-to-bastion:
+  securityGroup: securityGroup/bastion.{{ ClusterName }}
+  sourceGroup: securityGroup/nodes.{{ ClusterName }}
+securityGroupRule/all-master-to-bastion:
+  securityGroup: securityGroup/bastion.{{ ClusterName }}
+  sourceGroup: securityGroup/masters.{{ ClusterName }}
+securityGroupRule/ssh-external-to-bastion:
+  securityGroup: securityGroup/bastion.{{ ClusterName }}
+  sourceGroup: securityGroup/bastion-elb.{{ ClusterName }}
+  protocol: tcp
+  fromPort: 22
+  toPort: 22
+
+
+# ---------------------------------------------------------------
+# Security Group - Bastion ELB
+#
+# The security group that the bastion lives in
+# ---------------------------------------------------------------
+securityGroup/bastion-elb.{{ ClusterName }}:
+  vpc: vpc/{{ ClusterName }}
+  description: 'Security group for bastion ELB'
+  removeExtraRules:
+  - port=22
+securityGroupRule/bastion-elb-egress:
+  securityGroup: securityGroup/bastion-elb.{{ ClusterName }}
+  egress: true
+  cidr: 0.0.0.0/0
```

Comment:
I was guessing that after I wrote the comment.  There is a configuration value that sets the ip range of base sec group.  Lets talk when I am not as tired.